pcsx2/IopBios.cpp

```diff
@@ -611,6 +611,21 @@ namespace R3000A
 			return 0;
 		}
 
+		int remove_HLE()
+		{
+			const std::string full_path = clean_path(Ra0);
+
+			if (is_host(full_path))
+			{
+				const std::string path = full_path.substr(full_path.find(':') + 1);
+				ghc::filesystem::path file_path{host_path(path)};
+				bool suceeds = ghc::filesystem::remove(file_path);
```

Comment:
Can also be marked as const?

Reply:
Should we do it then in the rest of places in this file?

Reply:
sure, why not :)

Reply:
Done!